src/EditorFeatures/Core/Implementation/LineSeparators/AdornmentManager.cs

```diff
@@ -302,6 +316,29 @@ private void UpdateInvalidSpans()
             }
         }
 
+        private NormalizedSpanCollection GetCollapsedSpans()
+        {
+            var elisionSnapshot = _textView.VisualSnapshot as IProjectionSnapshot;
```

Comment:
@Pilchie: I think IProjectionSnapshot is used for both. You might be thinking that IProjectionBuffer and IElisionBuffer both derive from IProjectionBufferBase.